CoffeeShop.js

```diff
@@ -0,0 +1,106 @@
+class File {
+	static Exists(filePath) {
+		if (require("fs").existsSync(filePath))
+			return true;
+
+		return false;
```

Comment:
https://medium.com/@ntgard/returning-boolean-literals-is-a-code-smell-7a39531d6b60

Reply:
Thanks for the article!